<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleClient</class>
 <widget class="QMainWindow" name="ExampleClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExampleClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="introPage">
       <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,1" columnstretch="1,0,1">
        <item row="1" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QStackedWidget" name="connectionGroup">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="verticalStackedWidgetPage1">
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Server:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="connectionBox">
                <property name="editable">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>ws://localhost:50000</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Client Name:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="clientNameEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="connectServer">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="maximum">
               <number>0</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="runningPage">
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="text">
                <string>...</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="viewHolder" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="tabPosition">
             <enum>QTabWidget::West</enum>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="sceneTab">
             <attribute name="title">
              <string>Scene</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,5">
              <item>
               <widget class="QTreeWidget" name="objectTreeWidget">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <column>
                 <property name="text">
                  <string>Object</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="title">
                   <string>Object Methods</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <property name="leftMargin">
                    <number>3</number>
                   </property>
                   <property name="topMargin">
                    <number>3</number>
                   </property>
                   <property name="rightMargin">
                    <number>3</number>
                   </property>
                   <property name="bottomMargin">
                    <number>3</number>
                   </property>
                   <item>
                    <widget class="QListView" name="objAttMethodsList"/>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="title">
                   <string>Object Tables</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="leftMargin">
                    <number>3</number>
                   </property>
                   <property name="topMargin">
                    <number>3</number>
                   </property>
                   <property name="rightMargin">
                    <number>3</number>
                   </property>
                   <property name="bottomMargin">
                    <number>3</number>
                   </property>
                   <item>
                    <widget class="QListView" name="objAttTablesList"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="documentTab">
             <attribute name="title">
              <string>Document</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_14">
              <item>
               <widget class="QListView" name="documentMethodView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tablesTab">
             <attribute name="title">
              <string>Tables</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QTableView" name="tableView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="methodTab">
             <attribute name="title">
              <string>Methods</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="10,0,3">
              <item>
               <widget class="QTableView" name="methodsView">
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="methodNameLabel">
                <property name="text">
                 <string>Method Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="methodDescBox"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="signalTab">
             <attribute name="title">
              <string>Signals</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QTableView" name="signalsView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="lightTab">
             <attribute name="title">
              <string>Lights</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QTableView" name="lightsView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="materialsTab">
             <attribute name="title">
              <string>Materials</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QTableView" name="materialsView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="meshesTab">
             <attribute name="title">
              <string>Meshes</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QTableView" name="meshesView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="texturesTab">
             <attribute name="title">
              <string>Textures</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QTableView" name="texturesView"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="buffersTab">
             <attribute name="title">
              <string>Buffers</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_12">
              <item>
               <widget class="QTableView" name="buffersView"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
